07 Form/src/components/member/pageContainer.tsx

```diff
@@ -50,8 +54,8 @@ export class MemberPageContainer extends React.Component<{}, State> {
       <MemberPage
         member={this.state.member}
         onChange={this.onFieldValueChange}
-        onSave={this.onSave}
+        onSave={this.onSave(this.props)}
```

Comment:
not necessary passing props as argument because the `onSave` method is define inside class.

You could:

```javascript
private onSave = () => {
  ....
  this.props.history.goback();
....
}
```